pom.xml

```diff
@@ -352,5 +357,27 @@
         <javadoc.opts>-Xdoclint:none</javadoc.opts>
       </properties>
     </profile>
+    <profile>
+      <id>regression-tests</id>
```

Comment:
I don't really fancy the name `regression-tests` prefer `itests` which is more descriptive.